website/blog/2022-05-10-faster-presto-queries-with-parquet-page-index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Faster Presto Queries with Parquet Page Index
+author: Xinli Shang
+authorURL: https://www.linkedin.com/in/xinli-shang-2479863/
+---
+**Uber:** Xinli Shang
+
+**Uber:** Chen Liang
+
+## Introduction
+Today’s data is growing very fast, which creates challenges for query engines like Presto.  Presto is a popular interactive query engine, because of its scalability, high performance, and smooth integration with Hadoop. As the volume of data grows, Presto needs to read larger chunks of data and load them into memory, which causes higher IO and memory usage and GC time etc.
```

Comment:
```suggestion
Today’s data is growing very fast, which creates challenges for query engines like Presto.  Presto is a popular interactive query engine, because of its scalability, high performance, and smooth integration with Hadoop. As the volume of data grows, Presto needs to read larger chunks of data and load them into memory, which causes higher IO, memory usage, and GC time etc.
```